lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,28 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    // Set digital object URL
+    $digitalObjects = $this->getDigitalObjects();
+    if (count($digitalObjects) > 0)
+    {
+      $do = $this->digitalObjects[0];
+      $checksum = $do->getChecksum();
+
+      return $checksum;
+    }
+    else
```

Comment:
Else isn't really needed here. You can just return null after the if. Also, if you don't explicitly return at all, PHP returns null by default in functions. Very minor point though, I think this is fine.